### DISCUSSION TITLE
:popover-open and Arbitrary Properties

### DISCUSSION THREAD

**Discussion by cjmellor** [tailwindlabs/tailwindcss#13568] [Help]
Hi,

I know Tailwind doesn't support the [popover-open pseudo class](https://developer.mozilla.org/en-US/docs/Web/CSS/:popover-open) yet, but I thought with [arbitrary properties](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) I might be able to use it, but I'm not sure I am doing it right, as I can't seem to get it to work.

```html
[popover-open:inset-0]
```

or 

```html
[popover-open:inset-[unset]]
```

Any thoughts?

Thanks.

**Comment by andymccray** [comment]
`[popover-open:inset-0]` will compile to an actual CSS property and `inset-0` would be an invalid value.

You can create an arbitrary pseudo-class, e.g. `[&:popover-open]:inset-0` and that should do the trick.

Tailwind Play example: https://play.tailwindcss.com/nfMNm6Z0i2 Open the generated CSS to see what the former produces.

**Reply by cjmellor** [reply]
Ahh, I was close then 😅

Thanks!
